content/docs/studio/user-guide/model-registry/register-version.md

```diff
@@ -1,14 +1,29 @@
-# Version models
+# Register a model version
```

Comment:
Are there any links to this section from anywhere else in the docs/studio? If yes, they'll need to be updated.

Reply:
Same as above. If a page has multiple sections, sometimes we also use the section url specifically (such as `doc/studio/user-guide/model-registry/register-version#version-models`). Maybe there's none for this section, but would be good to confirm.

Reply:
Confirmed.